Documentation/Troubleshooting/ceph-common-issues.md

```diff
@@ -774,3 +777,36 @@ data: {}
 ```
 
 If the ConfigMap exists, remove any keys that you wish to configure through the environment.
+
+## The cluster is in an unhealthy state or fails to configure
```

Comment:
Is there any more specific title that would help describe this issue?

Reply:
Issue was pretty generic IMHO, hard to troubleshoot and the main symptoms are the ones described in the title. But I think I can be a little more specific.